modules/nodes-cluster-enabling-features-about.adoc

```diff
@@ -19,15 +19,26 @@ The following Technology Preview features are enabled by this feature set:
 ** CSI automatic migration. Enables the automatic migration of supported in-tree volume plug-ins to their equivalent Container Storage Interface (CSI) drivers. Available as a Technology Preview for:
 *** Amazon Web Services (AWS) Elastic Block Storage (EBS)
 *** OpenStack Cinder
+*** Azure Disk
```

Comment:
Is this right?

Reply:
That's how it's listed in `main` if I am reading that file right

Reply:
Right, but do you know if it's valid in 4.8? I suggest not making this branch match main here.